### ISSUE TITLE
Add connection type to $env

### ISSUE THREAD

**Issue by gliechtenstein** [Jasonette/documentation#67]
We could detect if it's wifi or 3g or lte, etc. and render completely different versions of an app if we wanted. Just need to add another attribute to [$env](http://docs.jasonette.com/env/)

**Comment by maks** [comment]
could this also include 'offline' to allow detecting that state too?

**Comment by gliechtenstein** [comment]
@maks yes, i can't confirm util i try but i presume that should be feasible too!
